pallets/gear/src/benchmarking/code.rs

```diff
@@ -371,12 +371,24 @@ where
     }
 
     pub fn unary_instr(instr: Instruction, repeat: u32) -> Self {
```

Comment:
Can rename this into unary_instr_64 if it would look better

Reply:
sure